### DISCUSSION TITLE
Adobe AIR error: there was a problem parsing the package/

### DISCUSSION THREAD

**Discussion by ZhorovEgor** [airsdk/Adobe-Runtime-Support#3077] [General]
Howdy everyone! I had a problem with compile project in Adobe Animate AIR by Harman.
I want to compile a project for Android (API version 34), but after connecting the necessary connections in the configuration file, I end up with a broken application. Connecting the SDK is due to the fact that the output file must be in AAB format, I did as was necessary in one of the videos, double-checked and set the SDK versions I needed through Android Studio, and if I compile the project without setting adt.cfg, then the application is in the format The APK is created correctly. Tell me who is familiar with the problem, I am attaching screenshots below.

Video instruction: https://www.youtube.com/watch?v=RX0cJW3BuxA

My SDK
![Screenshot_1](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/e11d61c1-8c7d-4eb4-89f3-ba9655996790)

adt.cfg setting
![Screenshot_4](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/2edb7332-5e70-469f-8967-8686c5d08a7e)

after compile project with adt.cfg
![Screenshot_3](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/817dcb3e-224b-4b4d-8811-43555545655b)

Run install in environment 
![Screenshot_2](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/3db8b408-7df8-4f22-b555-4ac7ab000e6c)

**Comment by ajwfrost** [comment]
Hi - I can't see anything wrong with any of the set-up there. The warning message is to be expected because there are multiple "build-tool" folders but we just use the latest available one.
Would it be possible to get the resulting package so that we can try to load it up here, and to unzip/inspect it to see why you're getting this message?

Or also, just thinking about it, given you're using Animate:
- it says that the output filename is "Wizdom.apk"
- can you please try renaming that - on the file system, i.e. after Animate has finished - to "Wizdom.aab"
- and then can you open the AAB file within BlueStacks?

If you can't open an AAB file in BlueStacks then you need to comment out this line:
```
# CreateAndroidAppBundle=true
```
Because with this, it's generating a bundle (aab) file, but Animate doesn't know about this and so still asks for it to be called something.apk. If BlueStacks recognises file types purely based on the extension, this could be the reason for their error message....

thanks

**Reply by balaedit** [reply]
BlueStacks support only apk till yet. We needs to comment using `#CreateAndroidAppBundle=true` and publish to test it on BlueStacks app. Apart from this, you need to test this using Animate CC provide debugg via usb. Animate CC show the BlueStacks device in debugger. You can test live there even if `CreateAndroidAppBundle=true` line is uncommented. I tested it on BlueStacks and Android Studio Virtual Device.

**Reply by ZhorovEgor** [reply]
""it says that the output filename is "Wizdom.apk""" - If i put sone other filename (Wizdom.aab), For Animate It does not matter, he will create "Wizdom.aab.apk" file.
""can you please try renaming that - on the file system, i.e. after Animate has finished - to "Wizdom.aab""" - answer on top.
""and then can you open the AAB file within BlueStacks?"" - I can't open aab file, anywhere, couse only Google Console use this format

**Reply by ZhorovEgor** [reply]
I upload the project, can you chek it please? https://drive.google.com/file/d/1RpcEP5lVkKzvC307WAehvEKnSO0hFjVu/view?usp=sharing

**Reply by ajwfrost** [reply]
I can't download that (IT policy blocks Google Drive) but if you do have an AAB file, you can either convert this to APK files using the Google 'bundletool', or you can install that onto an emulator using the ADT command line. I'm not quite sure whether ADB works with BlueStacks though? When it's running, try on a command line from the AIR SDK "bin" folder:
```
adt -devices -platform android
```
If it lists the BlueStacks emulator then you can do
```
adt -install -package Wizdom.apk
```
and even if it's actually an AAB file, we should do the conversion and install it as an APK.

thanks

**Reply by ZhorovEgor** [reply]
https://drive.google.com/file/d/1-2BNT2UqZ_Ga9N7i2ScvDXGrxgw_WYvX/view?usp=sharing 

I refreshed link password: 123

**Comment by ZhorovEgor** [comment]
Good afternoon Over the weekend I tried to comment out the text "#CreateAndroidAppBundle=true" and ended up getting a message from Animate:

![Screenshot_1](https://github.com/airsdk/Adobe-Runtime-Support/assets/63095086/f9e5fd5b-08c7-4139-bf91-847743b1e6eb)
